src/Parser/DestructionWarlock/Modules/SoulShards/SoulShardDetails.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+import Tab from 'Main/Tab';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+import SoulShardBreakdown from './SoulShardBreakdown';
+import SoulShardTracker from './SoulShardTracker';
+
+import WastedShardsIcon from '../../Images/affliction_warlock_soulshard_bw.jpg';
+
+const soulShardIcon = 'inv_misc_gem_amethyst_02';
+
+class SoulShardDetails extends Module {
+  static dependencies = {
+    soulShardTracker: SoulShardTracker,
+  };
+
+  suggestions(when) {
+    const shardsWasted = this.soulShardTracker.shardsWasted;
+    when(shardsWasted).isGreaterThan(0)
```

Comment:
If possible please try to use percentage or per minute numbers for triggers so that something like wasting 10 on a 3 minute fight can get a higher prio than wasting 10 on a 15 minute fight. I recently wrote guidelines for suggestions, here: https://github.com/MartijnHols/WoWAnalyzer/blob/master/CONTRIBUTING.SPEC.md#suggestions

Reply:
My situation with this is pretty much identical to Iskalla's problem, and I'm torn apart as to how it should be. I agree with you that it should be somewhat proportionate to the fight length, but also that really no shard should be wasted regardless of the fight length. I'll try to change it but the thresholds will be tricky for both.